Content.Server/Maps/TileSystem.cs

```diff
@@ -54,6 +55,22 @@ public bool CutTile(TileRef tileRef)
         return DeconstructTile(tileRef);
     }
 
+    public bool ReplaceTile(TileRef tileref, ContentTileDefinition replacementTile)
+    {
+        var variant = _robustRandom.Pick(replacementTile.PlacementVariants);
+
+        if (!TryComp<MapGridComponent>(tileref.GridUid, out var grid))
+            return false;
+
+        var decals = _decal.GetDecalsInRange(tileref.GridUid, tileref.GridPosition().SnapToGrid(EntityManager, _mapManager).Position, 0.5f);
```

Comment:
Not looking at my IDE but does this work for larger tile sizes?

Reply:
this is what crowbars use for removing decals on pry. If this doesn't work for larger decals, then that's probably an issue with decals being kinda funny. Sorry if you meant something else by "larger tile sizes"